cicd-scripts/customize-mco.sh

```
@@ -57,6 +58,10 @@ update_mco_cr() {
     "${SED_COMMAND[@]}" "/annotations.*/a \ \ \ \ mco-multicluster_observability_addon-image: ${MULTICLUSTER_OBSERVABILITY_ADDON_IMAGE_REF}" ${ROOTDIR}/examples/mco/e2e/v1beta2/observability.yaml
     "${SED_COMMAND[@]}" "/annotations.*/a \ \ \ \ mco-multicluster_observability_addon-image: ${MULTICLUSTER_OBSERVABILITY_ADDON_IMAGE_REF}" ${ROOTDIR}/examples/mco/e2e/v1beta2/*/observability.yaml
   fi
+  if [[ -n ${GRAFANA_IMAGE_REF} ]]; then
+    "${SED_COMMAND[@]}" "/annotations.*/a \ \ \ \ mco-grafana-image: ${GRAFANA_IMAGE_REF}" ${ROOTDIR}/examples/mco/e2e/v1beta2/observability.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)